adoc/chapters/programming_interface.adoc

```diff
@@ -4917,6 +4916,9 @@ info::event_command_status::complete
 ----
    a@ Indicates that the command has finished running on the device.
       Attempting to wait on such an event will not block.
+      
+      Synchronization: When event::get_info returns this value,
+      the synchronization is equivalent to event::wait.
```

Comment:
Please add `[code]#...#` decoration.

Reply:
Do you want `code` or `api`?

Reply:
`api` only works for the classes that have been reformatted into the new style (because there needs to be a matching `apidef` whenever you use `api`).  The `event` class hasn't been reformatted yet, so you need to use `code` here.